packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js

```diff
@@ -151,41 +154,293 @@ describe('ContentCards', () => {
         });
     });
-});
 
+    describe('when given 2 adapters that both return cards', () => {
+        let instance;
+        let adapter;
+        let adapter2;
+        let spy;
+        let spy2;
+        let adapterInstance;
+        let adapterInstance2;
+        const mockCards = [
+            {
+                id: '__MOCK_CARD__',
+                type: '__MOCK_TYPE__'
+            }
+        ];
+        const mockCards2 = [
+            {
+                id: '__MOCK_CARD_2__',
+                type: '__MOCK_TYPE_2__'
+            }
+        ];
+        const mockFilter = jest.fn();
 
-// when given 2 adapters that both return cards
+        beforeEach(() => {
+            // Arrange
+            adapter = () => ({
+                source: 'mockAdapter',
+                initialise: (_, callback) => {
+                    callback(mockCards);
+                },
+                handleCardView: jest.fn(),
+                handleCardClick: jest.fn()
+            });
+            adapterInstance = adapter();
 
-// it should combine both adapters cards by pushing cards into cards array
+            adapter2 = () => ({
+                source: 'mockAdapter',
+                initialise: (_, callback) => {
+                    callback(mockCards2);
+                },
+                handleCardView: jest.fn(),
+                handleCardClick: jest.fn()
+            });
+            adapterInstance2 = adapter2();
 
-// it should return cards from both adapters in the scoped slot variable
+            spy = jest.spyOn(adapterInstance, 'initialise');
+            spy2 = jest.spyOn(adapterInstance2, 'initialise');
+
+            instance = shallowMount(ContentCards, {
+                propsData: {
+                    adapters: [adapterInstance, adapterInstance2],
+                    filters: [mockFilter],
+                    locale: 'uk'
+                },
+                scopedSlots,
+                mocks
+            });
+        });
+
+        it('should push cards returned from adapter into cards', () => {
+            // Act
+            const { cards } = instance.vm;
 
-// it should return the default slots state
+            // Assert
+            expect(cards.length).toEqual(2);
+        });
 
-// it should emit event to indicate each adapter has returned cards
+        it('should return the default slots state', () => {
+            // Act
+            const slot = instance.find(`[data-test-id="slot-${STATE_DEFAULT}"]`);
 
-// it should filter cards for each adapter based on adapter assigned filters
+            // Assert
+            expect(slot).toBeDefined();
+        });
 
+        it('should return these cards in the scoped slot variable', () => {
+            // Act
+            const cards = instance.findAll('[data-test-id="__MOCK_TYPE__"]');
+            const cards2 = instance.findAll('[data-test-id="__MOCK_TYPE_2__"]');
 
+            // Assert
+            expect(cards.length + cards2.length).toEqual(2);
```

Comment:
Might it be better to have two expectations?
```suggestion
            expect(cards).toHaveLength(1);
            expect(cards2).toHaveLength(1);
```